mysite/settings.py

```diff
@@ -52,6 +52,7 @@
         'DIRS': [],
         'APP_DIRS': True,
         'OPTIONS': {
+            'debug': True,
```

Comment:
Ideally, this shouldn't be the default, however it's fine to add a "todo" for now:
```suggestion
            'debug': True,  # FIXME: Only enable this during testing
```

Reply:
> Ideally, this shouldn't be the default, however it's fine to add a "todo" for now:

I deleted `FIXME:` because:
```
pylint...................................................................Failed
- hook id: pylint
- exit code: 4

************* Module mysite.settings
mysite/settings.py:55:29: W0511: FIXME Only enable this during testing (fixme)
```
I didn't find a solution to how I can keep this comment about the fix, in` .pylintrc` :

```
[MISCELLANEOUS]

# List of note tags to take in consideration, separated by a comma.
notes=FIXME,
      XXX,
      TODO
```